lib/order.rb

```diff
@@ -33,6 +33,13 @@ def check_order_prompt
     "Please check your order against your total:"
   end
 
```

Comment:
did I understand User Story 3 correctly or did I go overboard?
```
As a customer
So that I can verify that my order is correct
I would like to check that the total I have been given matches the sum of the various dishes in my order'

```
did I satisfy this condition simply by giving the customer a 'total' method?